### DISCUSSION TITLE
keep-your-users-informed-with-asp-net-mvc/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#41] [Blog comments]
# keep-your-users-informed-with-asp-net-mvc/

https://blogs.taiga.nl/martijn/posts/keep-your-users-informed-with-asp-net-mvc/

**Comment by disqus-comments-migrator** [comment]
<em>Peter Bos commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at mei 13 2011, 03:22</em>

---

<p>works like a charm, thankz!</p>

**Comment by disqus-comments-migrator** [comment]
<em>Kevin Bolander commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at januari 23 2012, 02:28</em>

---

<p>Thank you sir, that was hella helpful.</p>

**Comment by disqus-comments-migrator** [comment]
<em>Paul Reynolds commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at februari 02 2012, 03:31</em>

---

<p>A really good post; very helpful and a nice neat minimal solution :-)</p>

**Comment by disqus-comments-migrator** [comment]
<em>Mohammad Asghari commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at maart 11 2012, 11:37</em>

---

<p>That you very much! that was excellent!</p>

**Comment by disqus-comments-migrator** [comment]
<em>Deepak commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at maart 21 2012, 06:18</em>

---

<p>Great Post!</p>

**Comment by disqus-comments-migrator** [comment]
<em>JP Negri commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at april 27 2012, 05:10</em>

---

<p>Thanks! Great post, great code!</p>
